gdk-pixbuf/src/auto/pixbuf_animation.rs

```diff
@@ -137,7 +137,12 @@ impl PixbufAnimation {
     }
 }
 
-pub trait PixbufAnimationExt: IsA<PixbufAnimation> + 'static {
+mod sealed {
```

Comment:
This is blocked by someone updating all the manual traits accordingly (CC @jf2048 )